Binary files tables-3.1.1/dist/tables-3.1.1.win-amd64-py2.7.exe and tables-3.1.1-new/dist/tables-3.1.1.win-amd64-py2.7.exe differ
diff -ruN tables-3.1.1/tables/__init__.py tables-3.1.1-new/tables/__init__.py
--- tables-3.1.1/tables/__init__.py	2015-04-19 09:25:51.191872000 -0600
+++ tables-3.1.1-new/tables/__init__.py	2014-03-25 14:49:03.000000000 -0600
@@ -60,9 +60,9 @@
     # In order to improve diagnosis of a common Windows dependency
     # issue, we explicitly test that we can load the HDF5 dll before
     # loading tables.utilsextensions.
-    if not _load_library('hdf5.dll', ctypes.cdll.LoadLibrary):
+    if not _load_library('hdf5dll.dll', ctypes.cdll.LoadLibrary):
         raise ImportError(
-            'Could not load "hdf5.dll", please ensure'
+            'Could not load "hdf5dll.dll", please ensure'
             ' that it can be found in the system path')
 
     # Some PyTables binary distributions place the dependency DLLs in the
